src/pages/partners/affiliate-ib/_dmt5-cards.tsx

```diff
@@ -76,33 +76,82 @@ const query = graphql`
+        mt5_forex: file(relativePath: { eq: "affiliate-and-ib/mt5_forex.png" }) {
+            ...fadeIn
+        }
+        mt5_metal: file(relativePath: { eq: "affiliate-and-ib/mt5_metal.png" }) {
+            ...fadeIn
+        }
+        mt5_oil: file(relativePath: { eq: "affiliate-and-ib/mt5_oil.png" }) {
+            ...fadeIn
+        }
+        mt5_crypto: file(relativePath: { eq: "affiliate-and-ib/mt5_crypto.png" }) {
+            ...fadeIn
+        }
+        mt5_stocks: file(relativePath: { eq: "affiliate-and-ib/mt5_stocks.png" }) {
+            ...fadeIn
+        }
+        mt5_stock_indices: file(relativePath: { eq: "affiliate-and-ib/mt5_stock_indices.png" }) {
+            ...fadeIn
+        }
+        mt5_volatility: file(relativePath: { eq: "affiliate-and-ib/mt5_volatility.png" }) {
+            ...fadeIn
+        }
+        mt5_step: file(relativePath: { eq: "affiliate-and-ib/mt5_step.png" }) {
+            ...fadeIn
+        }
+        derived_crash: file(relativePath: { eq: "affiliate-and-ib/derived_crash.png" }) {
+            ...fadeIn
+        }
+        derived_jump: file(relativePath: { eq: "affiliate-and-ib/derived_jump.png" }) {
+            ...fadeIn
+        }
+        derived_step: file(relativePath: { eq: "affiliate-and-ib/derived_step.png" }) {
+            ...fadeIn
+        }
+        derived_volatility: file(relativePath: { eq: "affiliate-and-ib/derived_volatility.png" }) {
+            ...fadeIn
+        }
+        fin_crypto: file(relativePath: { eq: "affiliate-and-ib/fin_crypto.png" }) {
+            ...fadeIn
+        }
+        fin_forex: file(relativePath: { eq: "affiliate-and-ib/fin_forex.png" }) {
+            ...fadeIn
+        }
+        fin_metal: file(relativePath: { eq: "affiliate-and-ib/fin_metal.png" }) {
+            ...fadeIn
+        }
+        fin_oil: file(relativePath: { eq: "affiliate-and-ib/fin_oil.png" }) {
+            ...fadeIn
+        }
+        fin_stock: file(relativePath: { eq: "affiliate-and-ib/fin_stock.png" }) {
+            ...fadeIn
+        }
+        stp_forex: file(relativePath: { eq: "affiliate-and-ib/stp_forex.png" }) {
+            ...fadeIn
+        }
+        stp_crypto: file(relativePath: { eq: "affiliate-and-ib/stp_crypto.png" }) {
+            ...fadeIn
+        }
     }
 `
 
 const StyledCard = styled(Card)`
-    min-height: 42.6rem;
-    height: 100%;
+    min-block-size: 42.6rem;
+    block-size: fit-content;
+    min-inline-size: calc((100% - 4.8rem) / 3);
+    inline-size: calc((100% - 4.8rem) / 3);
     position: relative;
     border-radius: 0.8rem;
-    min-width: calc((100% - 4.8rem) / 3);
-    width: calc((100% - 4.8rem) / 3);
     justify-content: flex-start;
 
-    :nth-child(2) {
-        margin: 1.6rem 2.4rem 0;
-
-        @media ${device.tabletL} {
-            margin: 2.4rem 0;
-        }
-    }
-
     @media ${device.laptopM} {
-        min-width: 38.4rem;
-        width: 38.4rem;
+        min-inline-size: 38.4rem;
+        inline-size: 38.4rem;
```

Comment:
Is both `min-inline-size` and` inline-size` needed?

Reply:
no, just renamed the existing styles correctly, remove then

Reply:
remove